web/src/components/netflow-traffic.tsx

```diff
@@ -127,6 +130,15 @@ export const NetflowTraffic: React.FC<{
     [getQueryArguments, selectedViewId]
   );
 
+  const fetchConfig = React.useCallback(() => {
+    getConfig()
+      .then(setConfig)
+      .catch(err => {
+        const errorMessage = getHTTPErrorDetails(err);
+        console.log(errorMessage);
+      });
+  }, [setConfig]);
```

Comment:
Remove the `useCallback` here so it will be called only once

If you want a refresh strategy we can call that function again on some actions like:
- manage columns popup click
- specific delay
- ticks (but not all)